.github/workflows/release.yml

```diff
@@ -16,14 +16,20 @@ jobs:
     runs-on: ubuntu-latest
     name: 'Release a new version'
     steps:
+      - name: Get auth token
+        id: token
+        uses: actions/create-github-app-token@5d869da34e18e7287c1daad50e0b8ea0f506ce69 # v1.11.0
```

Comment:
Sidenote: Does this do the same thing as https://github.com/getsentry/action-github-app-token (if so, can we archive?)

Reply:
It does! It's actually more up-to-date on the github library it uses and a bit more feature rich, so yes, but we do need to update all the workflows that uses it before we can archive it.